### PR TITLE
Bugfix/3328 rest client no charset option

### PR DESCRIPTION
**Pull request by turlyalz**

### CODE REVIEW

File: qlib/RestClient.qm [hunk 1]
```diff
@@ -124,6 +124,9 @@ printf("%N\n", ans.body);
 
     @section restclientrelnotes Release Notes
 
+    @subsection restclientv1_7 RestClient v1.7
```
Comment (davidnich) [inline qlib/RestClient.qm]: this should be 1.6.1 for an intermediate release

**Review by davidnich — COMMENTED** [review]
looks good - just one minor change + please add a release note in `doxygen/lang/900_release_notes.dox.tmpl` as well

File: qlib/RestClient.qm (hunk 1 above)
Comment (turlyalz) [reply qlib/RestClient.qm]: thanks, fixed :)

File: doxygen/lang/900_release_notes.dox.tmpl [hunk 2]
```diff
@@ -15,6 +15,8 @@
       deserializing lists with complex type information
 
     @subsection qore_091_bug_fixes Bug Fixes in Qore
+    - fixed a bug with charset in the RestClient module (issue 3328)
```
Comment (davidnich) [inline doxygen/lang/900_release_notes.dox.tmpl]: please leave a relnote for a module fix in the same format as the other modules - look a little below for lines like:
```
   - <a href="../../modules/Util/html/index.html">Util</a> module fixes:
      - fixed a bug in \c get_exception_string() with Java exceptions
        (<a href="https://github.com/qorelanguage/qore/issues/3304">issue 3304</a>)
```

**Review by davidnich — CHANGES_REQUESTED** [review]
one more small change please

File: doxygen/lang/900_release_notes.dox.tmpl (hunk 2 above)
Comment (davidnich) [reply doxygen/lang/900_release_notes.dox.tmpl]: and please put the module fix lines all together...

**Review by davidnich — APPROVED** [review]

**Pull request merged**